dashboard/app/models/reference_guide.rb

```diff
@@ -18,4 +18,70 @@
 #  index_reference_guides_on_course_version_id_and_parent_key  (course_version_id,parent_reference_guide_key)
 #
 class ReferenceGuide < ApplicationRecord
+  belongs_to :course_version
+  validates_uniqueness_of :key, scope: :course_version_id
+
+  def children
+    ReferenceGuide.where(course_version_id: course_version_id, parent_reference_guide_key: key)
+  end
+
+  def serialize
+    {
+      key: key,
+      course_version_key: course_version.key,
+      course_offering_key: course_version.course_offering.key,
+      parent_reference_guide_key: parent_reference_guide_key,
+      display_name: display_name,
+      content: content,
+      position: position
+    }
+  end
+
+  def write_serialization
+    return unless Rails.application.config.levelbuilder_mode
+    offering_and_version = "#{course_version.course_offering.key}_#{course_version.key}"
```

Comment:
Wondering if we should use a dash instead of a string here.  For example for csd. The course offering key will be 'csd' and the course version key will be '2022'. Something of the format `csd-2022` matches the course name where as `csd_2022` does not